runelite-api/src/main/java/net/runelite/api/Experience.java

```diff
@@ -173,4 +176,22 @@ public static int getCombatLevel(int attackLevel, int strengthLevel,
 	{
 		return (int) getCombatLevelPrecise(attackLevel, strengthLevel, defenceLevel, hitpointsLevel, magicLevel, rangeLevel, prayerLevel);
 	}
+
+	/**
+	 * Calculate the percentages of each level compared to the difference of the goal exp and the start exp
+	 *
+	 * @param startXp Starting exp
+	 * @param goalXp The goal exp
+	 */
+	public static List<Double> levelPercentagesBetweenExperience(int startXp, int goalXp)
```

Comment:
I would either move this logic to the xp globe plugin, or figure out how to reuse it in the xptracker since it already has similar logic.